src/js/libgeo/Tracing.js

```diff
@@ -140,7 +140,7 @@ function movepointscr(mover, pos, type) {
 /*
  * traceMover moves mover from current param to param for pos along a complex detour.
  */
-function traceMover(mover, pos, type) {
+function traceMover(mover, pos, type, knownTargetParam) {
```

Comment:
@strobelm The changes in [Tracing.js](https://github.com/strobelm/CindyJS/blob/CircleRadius/src/js/libgeo/Tracing.js) are obsolete. Please remove all occurences of `knownTargetParam` in `traceMover`.